frameworks/helloworld/src/main/dist/svc.yml

```diff
@@ -7,6 +7,10 @@ pods:
     count: {{HELLO_COUNT}}
     placement: {{HELLO_PLACEMENT}}
     container:
+      rlimits:
```

Comment:
I'm confused.  How are we still supporting both YAML forms?  Looks like we don't use it anymore in the `getContainerInfo` method below.